little-books/src/components/card/blog-card/blog-card.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { blogActions } from '../../../store/blog-store';
+import styles from './blog-card.module.scss';
+
+
+const BlogCard = ({title,type,details,photo,isSelected})=>{
+
+    const blogData = {
+        title:title,
+        details:details,
+        photo:photo,
+        type:type
+    }
+    const isEditModeOn = useSelector((state:any)=>state.editBlogState);
+
+    const dispatch = useDispatch();
+    const theme  = useSelector((state:any)=>state.theme);
```

Comment:
Why do we keep the theme state in redux?

Reply:
theme is been used by multiple components so its kept in redux